docs/ref/checks.txt

```diff
@@ -309,8 +309,6 @@ Related fields
 * **fields.W341**: ``ManyToManyField`` does not support ``validators``.
 * **fields.W342**: Setting ``unique=True`` on a ``ForeignKey`` has the same
   effect as using a ``OneToOneField``.
-* **fields.W343**: ``limit_choices_to`` has no effect on ``ManyToManyField``
-  with a ``through`` model.
```

Comment:
We don't want to reuse the same number in the future, so the docs should stay. Please add a note:
```
* **fields.W343**: ``limit_choices_to`` has no effect on ``ManyToManyField``
  with a ``through`` model. *This check appeared before Django 4.0.*
```